can/broadcastmanager.py

```diff
@@ -5,30 +5,39 @@
 :meth:`can.BusABC.send_periodic`.
 """
 
+import abc
+import logging
+import sys
+import threading
+import time
 from typing import Optional, Sequence, Tuple, Union, Callable, TYPE_CHECKING
 
+from typing_extensions import Final
+
 from can import typechecking
+from can.message import Message
 
 if TYPE_CHECKING:
     from can.bus import BusABC
 
-from can.message import Message
-
-import abc
-import logging
-import threading
-import time
 
 # try to import win32event for event-based cyclic send task (needs the pywin32 package)
+USE_WINDOWS_EVENTS = False
 try:
     import win32event
 
-    HAS_EVENTS = True
+    # Python 3.11 provides a more precise sleep implementation on Windows, so this is not necessary.
+    # Put version check here, so mypy does not complain about `win32event` not being defined.
+    if sys.version_info < (3, 11):
```

Comment:
Why do we event try to import win32event on Python 3.11+?

Reply:
Because i couldn't get mypy to shut up 👀